lib/rspec/core/example_group.rb

```diff
@@ -26,7 +26,7 @@ module Core
     # your examples defined in {MemoizedHelpers} and {Pending}.
     class ExampleGroup
       extend Hooks
-
+      extend FailFastHandler
```

Comment:
No apology needed.  Thanks for contributing!  RSpec is better because of contributions like yours.